force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -47,25 +48,14 @@ class psElevateTokenHandler {
             : `/apex/${TOKENIZE_CARD_PAGE_NAME}`;
     }
 
-    getVisualForceOriginURLs(domainInfo, namespace) {
+    async getVisualForceOriginURLs(domainInfo, namespace) {
         const url = `https://${domainInfo.orgDomain}--${namespace}.visualforce.com`;
```

Comment:
Not so sure if we need to keep this original format for the visual force URL to support older naming? Can remove if it's seen as unnecessary.

Reply:
@balsamhindi lets remove that `url` variable since we don't want to mess with any hardcoded urls anymore. You will also need to update some of the JEST tests that are looking for a urlOrigin length of 2

Reply:
That sounds good @daniel-fuller should I add the myDomain and lightning urls here or is the vforce enough? I think it should be since this page is meant for vf URLs but want to double check with you.

Reply:
@balsamhindi VF urls only if that is all we were handling for before

Reply:
Perfect!